Formula/mk-configure.rb

```diff
@@ -0,0 +1,22 @@
+class MkConfigure < Formula
+  desc "Lightweight replacement for GNU autotools"
+  homepage "https://github.com/cheusov/mk-configure"
+  url "https://downloads.sourceforge.net/project/mk-configure/mk-configure/mk-configure-0.29.3/mk-configure-0.29.3.tar.gz"
+  sha256 "1f483f717a09d06c82a6f3f284dd34234f54ead032e6987d33f3cf8109490c05"
+
+  depends_on "bmake"
+  depends_on "makedepend"
+
+  def install
+    ENV["PREFIX"] = prefix
+    ENV["MANDIR"] = man
+
+    system "bmake", "all"
+    system "bmake", "install"
+    doc.install "presentation/presentation.pdf"
+  end
+
+  test do
+    system "#{bin}/mkcmake", "-V", "MAKE_VERSION", "-f", "/dev/null"
```

Comment:
Wondering if we should just install the examples to `pkgshare` and then `cp_r` `pkgshare/"examples/hello_world"` and build/run that here.